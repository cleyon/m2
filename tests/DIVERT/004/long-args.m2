@newcmd foo{A}{B}{C}{D}
D=>@D@<
C=>@C@<
B=>@B@<
A=>@A@<
@endcmd
Begin
@foo{AA BB CC
DD EE FF}{GG HH II
JJ KK LL}{MM NN OO
PP QQ RR}{SS TT UU
VV WW XX}
End
