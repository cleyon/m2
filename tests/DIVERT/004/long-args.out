Begin
D=>SS TT UU
VV WW XX<
C=>MM NN OO
PP QQ RR<
B=>GG HH II
JJ KK LL<
A=>AA BB CC
DD EE FF<
End
